fuel/src/jsMain/kotlin/fuel/Fuel.kt

```diff
@@ -21,7 +21,7 @@ public actual object Fuel {
         httpLoader?.let { return it }
 
         // Create a new HttpLoader.
-        val loader = httpLoaderFactory?.newHttpLoader() ?: HttpLoader()
+        val loader = httpLoaderFactory?.newHttpLoader() ?: JSHttpLoader()
```

Comment:
Why is Fuel class still using expect/actual? Some of the methods don't vary at all across platforms. The only platform specific thing in here the lines like this one.

But you already have the platform specific mapping in FuelBuilder. It would seem to me that you could remove the expect/actual from the Fuel class and that line could just be:

```
        val loader = httpLoaderFactory?.newHttpLoader() ?: FuelBuilder().build()
```

Reply:
Done